internal/controllers/requirements/requirements.go

```diff
@@ -15,39 +15,28 @@
 package requirements
 
 import (
-	"errors"
-
-	"github.com/ZupIT/horusec-devkit/pkg/utils/logger"
-
 	"github.com/ZupIT/horusec/internal/controllers/requirements/docker"
 	"github.com/ZupIT/horusec/internal/controllers/requirements/git"
-	"github.com/ZupIT/horusec/internal/helpers/messages"
 )
 
-var ErrRequirements = errors.New("check the requirements for run and try again")
+type ValidationFn func() error
 
 type Requirements struct {
-	gitRequirements    *git.RequirementGit
-	dockerRequirements *docker.RequirementDocker
+	gitValidationFn    ValidationFn
+	dockerValidationFn ValidationFn
 }
 
 func NewRequirements() *Requirements {
 	return &Requirements{
-		gitRequirements:    git.NewRequirementGit(),
-		dockerRequirements: docker.NewRequirementDocker(),
+		gitValidationFn:    git.NewRequirementGit().ValidateGit,
+		dockerValidationFn: docker.NewRequirementDocker().ValidateDocker,
```

Comment:
We can use `git.ValidateGit` and `docker.ValidateDocker` directly, we don't need `RequirementGit` and `RequirementDocker` using this approach. 

I think that we can remove `Git` and `Docker` suffix for these functions since they already be on git and docker packages.